learn/contributing/contributing_to_docs.md

```diff
@@ -0,0 +1,91 @@
+- **Open source**. This is a resource by MeiliSearch users, for MeiliSearch users. [We welcome outside contributions](#how-to-contribute).
+
+## How to Contribute?
+
+### Issues
+
+On the MeiliSearch documentation team, [GitHub Issues](https://github.com/meilisearch/documentation/issues/new/choose) are the primary tool we use to track tasks. They're like elements in our to-do list. Here are some good uses for issues:
+
+- A detailed bug report
+- A request for a new technical feature (e.g. dark mode, versioning, etc.)
+- A request for new content (e.g. a tutorial, guide, or explanation)
+- A helpful suggestion for the future of the documentation
+
+We love issues, because they help us do our jobs better. Nine times out of ten, the most useful contribution isn't a flashy [pull request](#pull-requests) that needs to be read and reviewed—it's a simple GitHub issue that points out a problem and proposes a solution.
+
+#### Creating Your First Issue
+
+All you need to create an issue is a [GitHub account](https://github.com).
+
+1. Log into your account.
+2. Go to the MeiliSearch [Documentation repository](https://github.com/meilisearch/documentation).
+3. Click on "Issues".
+4. Click "New issue".
+5. Fill in a short, descriptive title and longer summary of your problem or suggestion.
+6. Click "Submit new issue".
+7. Enjoy the feeling of a job well done! 🎉
+
+### Pull Requests
+
+Sometimes, the problem is so small or particular that it would be faster for you to fix it yourself than explain it to us. In those cases, rather than create an [issue](#issues), you should make a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
+
+Pull requests ("PRs" for short) are requests to integrate your changes into an official repository. The simplest way to create a PR on our documentation is by using the **"Edit this page"** link at the bottom left of every page. [See below](#creating-your-first-pr) for more detailed instructions.
+
+Pull requests are particularly good for:
+
+- Fixing a small error, such as a typo, broken link, or poor word choice.
+- Making a change that you are especially qualified for (such as creating a guide for some software that you have mastered).
+- **Solving an [existing issue](https://github.com/meilisearch/documentation/issues)**!
+
+If the PR you're thinking about making isn't covered by the above list, it might be best to first [create an issue](#issues) to confirm that it's worth your time (and that nobody else is working on it already).
+
+#### Creating Your First PR
+
+There are many ways to create pull requests, but we'll cover the simplest method for this documentation site. Feel free to try it on this page!
+
+All you need to continue is a [GitHub account](https://github.com).
+
+1. On the documentation page you'd like to edit, scroll to the bottom and click **"Edit this page"** on the left. This will take you to GitHub.
```

Comment:
I think we want to keep these steps as simple as possible since it's for total beginners (plus the guide is designed for fixing typos and other small things, and those are unlikely to be sitting around as non-merged PRs 😄 )